cfme/tests/candu/test_utilization_metrics.py

```diff
@@ -36,6 +37,7 @@ def clean_setup_provider(request, provider):
 
 
 def vm_count(appliance, metrics_tbl, mgmt_system_id):
+    pytest.set_trace()
```

Comment:
Leftover from some debugging? Can this be removed?

Reply:
Yes, of course, that needs to go :)